pandas/io/formats/style.py

```diff
@@ -143,11 +143,16 @@ class Styler(StylerRenderer):
         uses ``pandas.options.styler.format.formatter``.
 
         .. versionadded:: 1.4.0
+    css : dict, optional
```

Comment:
why is this public facing?

Reply:
I think rather 'why not?'. When the original code developed for styler the css classes were hard coded. The docs describe a verbose way to go about changing these (if a web developer wants to). Having created the private class attribute here, for a purpose, it was a small simple extension to allow public facing, and it makes it much easier to customise the HTML.

Reply:
no what i mean is we have other things in a method, e.g. `set_table_styles`, so why not `set_css` (i know if we were to do this again maybe things should be *all* keywords of *all* methods but we have a mix now. want to rationalize this (keeping in mind signatures of other things and backwards compat).

Reply:
Ok, I get what you mean. With the exception of `cell_ids` all args to Styler instantiation can be configured with a `set_xxx` or `format` method currently.

With the exception of `uuid` all of these directly affect the visuals of the HTML so the user might want to adjust them, and typically does. 

`cell_ids` only reduces byte size and `css` won't have any direct affect unless the web developer has external css files, so not sure its worth adding a specific api method - its a pretty specific case use.